@here/harp-mapview/lib/Tile.ts

```diff
@@ -1288,6 +1290,9 @@ export class Tile implements CachedResource {
                 const bufferGeometry = new THREE.BufferGeometry();
 
                 srcGeometry.vertexAttributes.forEach((vertexAttribute: BufferAttribute) => {
+                    if (vertexAttribute.name === "color" && !hasVertexColors) {
```

Comment:
Can we skip generation of color vertex attribute in the decoded tile already (i.e. in OmvDecoder.getDecodedTile)?

Reply:
Good catch. I will check it out.